### PR TITLE
create components PageQuiz and Poster

### PR DESCRIPTION
**Pull request by JulienOza**

### CODE REVIEW

**Review by Rayane0823 — APPROVED** [review]
Reviewed

**Review by SebSamPro — APPROVED** [review]

File: frontend/src/App.jsx
Comment (Yoann-Fortin) [inline frontend/src/App.jsx:1]: What interest of div tag for one element ?

File: frontend/src/components/quiz-page/QuizPage.css
Comment (Yoann-Fortin) [inline frontend/src/components/quiz-page/QuizPage.css:1]: Why apply body style again ?

**Comment by JulienOza** [comment]
requested changes made

**Review by Rayane0823 — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Rayane0823 — APPROVED** [review]

**Review by Rayane0823 — APPROVED** [review]

**Pull request merged**